.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@7.2.1
+  hmpps: ministryofjustice/hmpps@7.3.4
```

Comment:
think dependabot may have introduced this very specific pinning - would probably better to just let it float at `7`.